pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
```

Comment:
Ideally the description should explain what the command does, not just what it is. The expanded name in this case is pretty self-descriptive, but it would be nice to maintain the same pattern found in other commands.